### ISSUE TITLE
Why the range of embedding make so much difference ?

### ISSUE THREAD

**Issue by yananchen1989** [keras-team/keras#10273]
I have tried many pre trained word embedding for my NN model. I find that when I initialize the embedding randomly with zero mean and 0.15 std, the training of NN goes fine. And if I initialize the embedding with a pre-trained model whose values all fall into [-1, 1], it also goes fine. But when I load the embedding where some values go beyond [-1, 1], like -12, -4, 8, 9 etc... the training of my NN model will get stuck within one epoch. The loss is swing between a value and it is stuck. If I use the model to predict, all the output will be zeros. So I am confused what's wrong with the embedding initialization ? Any solutions ?
